src/main/java/com/kodilla/ecommercee/dto/LoginDto.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.ecommercee.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+public class LoginDto {
+
+    private Boolean loggedIn;
```

Comment:
Jeżeli wartość ta będzie przyjmować tylko true / false, to lepiej użyć boolean. Boolean - może być null

Reply:
Fakt, że wartość wrappera może przybierać wartość `null` był właśnie powodem zastosowania typu `Boolean` - żeby mieć pewność, że wartość tego pola została ustawiona, a nie zainicjowana wartością domyślną. Choć przyznaję, że po zastanowieniu wydaje mi się, że akurat w tym miejscu niekoniecznie jest to istotne.